docs-ref-services/storage.md

```diff
@@ -14,49 +14,26 @@ ms.service: storage
 
 # Azure Storage modules for Node.js
 
-Use the Azure Storage client module to:
+Azure Storage is a Microsoft-managed service providing cloud storage that is highly available, secure, durable, scalable, and redundant. The following libraries in JavaScript make it easy to consume Microsoft Azure Storage service.
```

Comment:
Also, what's wrong with the old description? :)

Reply:
Hmmm.. let me clarify.
- Moved the older bullet list of descriptions to the table below.
- Copied this new description from our readme at azure-sdk-for-js repo.